coalib/bears/Bear.py

```diff
@@ -12,6 +12,7 @@
 from coala_utils.decorators import (enforce_signature, classproperty,
                                     get_public_members)
 
+# Ignore PyUnusedCodeBear
 from coalib.bears.requirements.PackageRequirement import PackageRequirement
 from coalib.bears.requirements.PipRequirement import PipRequirement
```

Comment:
PyUnusedCodeBear did not pick that up because it is used in the doc comment I think. But will do that.